interfaces/builtin/unity8_calendar.go

```diff
@@ -0,0 +1,130 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const unity8CalendarPermanentSlotAppArmor = `
+# Description: Allow operating as the EDS service. Reserved because this
+#  gives privileged access to the system.
+# Usage: reserved
+
+# DBus accesses
+#include <abstractions/dbus-session-strict>
+dbus (bind)
+	bus=session
+	name="org.gnome.evolution.dataserver.Calendar7",
+dbus (bind)
+	bus=session
+	name=org.gnome.evolution.dataserver.Subprocess.Backend.*,
```

Comment:
What is the '.*' referring to here? As written, it doesn't seem specific to calendar.